### ISSUE TITLE
Porting to python3

### ISSUE THREAD

**Issue by ColumOD** [architv/soccer-cli#146]
Hi, is this project still active? I am willing to help you with the porting to python3 as I have good experience doing this from other projects.

We should first make the code work in both python2 and 3 before making the full switch to python3. It would be good to get this done along with the addition of some new features before soccer returns!

Let me know if you are interested and I can get working on this asap!

**Comment by Saturn** [comment]
Yes, it would be good if this worked in Python 3. I can't imagine it being too much work. Maybe some of the messy code will cause some pain.

Back when I was working with this I was always using Python 2.7 just out of habit.

Of course, this used to be more active back when it was just started but now it has stagnated. But new features are welcome along with any refactoring that could make the code tidier etc.

Thanks,

**Comment by ColumOD** [comment]
Just setting up this now and some of the unit tests are failing. I am sure it is because of the fact there are currently no live games or fixtures and these exceptions are uncaught. Would you be able to quickly confirm they are not passing for you also before I go any further? Everything else is working just fine.

Thanks

**Comment by Saturn** [comment]
I think a lot (all) are failing and erroring because the code has changed without care of updating the unit tests to accompany the code change.

At least it shouldn't matter that there are no live games. Ideally.

I'll continue to have a look at it anyway.

**Comment by Shivanirudh** [comment]
Hello, so I have been trying to set this up and work. A peculiar thing i noticed was that i could not get any standings for UEFA Champions League. All other leagues work, and other options like fixtures, work for CL as well. Would you be able to confirm if you have this issue as well? Also, might this be because of the Champions League being divided into multiple groups? 

Thanks in advance.

**Comment by Saturn** [comment]
Hmm, CL standings does not work for me.

I can't rememeber if it did work in the past. It may not have ever had the CL standings, but just the fixtures.
